.evergreen/run-tests.sh

```diff
@@ -8,14 +8,14 @@ set -o errexit  # Exit the script with error if any of the commands fail
 #       UNIFIED                 Set to enable the Unified SDAM topology for the node driver
 #       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
 #       MARCH                   Machine Architecture. Defaults to lowercase uname -m
-#       TEST_NPM_SCRIPT         Script to npm run. Defaults to "check:test"
+#       TEST_NPM_SCRIPT         Script to npm run. Defaults to "check:coverage"
 #       SKIP_DEPS               Skip installing dependencies
 #       NO_EXIT                 Don't exit early from tests that leak resources
 
 AUTH=${AUTH:-noauth}
 UNIFIED=${UNIFIED:-}
 MONGODB_URI=${MONGODB_URI:-}
-TEST_NPM_SCRIPT=${TEST_NPM_SCRIPT:-check:test}
+TEST_NPM_SCRIPT=${TEST_NPM_SCRIPT:-check:coverage}
```

Comment:
This corresponds to `nyc npm run test:all` so the unit tests will be repeated on every variant. Maybe we want a new script for this to just run the int tests with coverage?

Reply:
Yeah, good catch!  Would we want a separate command to run our unit tests with coverage in CI as well?  (I'm assuming we run the unit tests in CI at some already...?)

Reply:
Yes we run them 🙂  they're part of the lint task, so if you go to that script you make it run nyc in front of the unit run there, and it would also get the upload after task just fine I think?